content/id/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -0,0 +1,281 @@
+---
+title: Init Container
+content_template: templates/concept
+weight: 40
+---
+
+{{% capture overview %}}
+Halaman ini menyediakan ikhtisar untuk Init Container, yaitu Container khusus yang dijalankan sebelum Container aplikasi dan berisi skrip peralatan atau _setup_ yang tidak tersedia di dalam _image_ dari Container aplikasi.
```

Comment:
all of the "container" word in the translated doc is translated to "kontainer". however, this page mostly talks about `pod.beta.kubernetes.io/init-containers`

for this page, do you think we should keep all words "Container" or translate them to "Kontainer"? If so, do we keep "Init Container" or it becomes "Kontainer Init"? Let me know what you think

IMO we should make it consistent with the rest of pages (i.e. Kontainer), but not sure if it will confuse people reading this page

cc @irvifa

Reply:
I think using "Container" would be better, as many Kubernetes API objects and specification related to containers are in form of the word "container".

It's even an API object in the core workload:
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.14/#container-v1-core

What do you think? @girikuncoro @irvifa

Reply:
So mostly we'll use Container when referring to the Container in Kubernetes.

We might also use "kontainer" when referring containers not related to Kubernetes.. 
or.. 
we can also stick with "container" to avoid confusion (even if the "kontainer" as in LXC cgroup mechanism, is added to kbbi as a tech noun in the future. Kontainer is already in the kbbi, but it actually refers to the cargo box)

Reply:
> I think mostly we'll use Container when referring to the Container in Kubernetes.
> We might also use "kontainer" when referring containers not related to Kubernetes..

I think this makes sense, we make default "kontainer" to our normal vocab, but if we talk about Container in Kubernetes I agree with you (this is aligned to the style guide, we treat Container as API resource with camel case, similar to Pod or Deployment)